pycalphad/core/eqsolver.pyx

```diff
@@ -112,7 +112,7 @@ cdef bint add_new_phases(composition_sets, phase_records, current_grid, chemical
     if largest_df > minimum_df:
```

Comment:
Should like 418 also use `np.ascontiguousarray` or continue to use `asfortranarray`?